cmake/CPM.cmake

```diff
@@ -156,6 +156,8 @@ set(CPM_SOURCE_CACHE
     CACHE PATH "Directory to download CPM dependencies"
 )
 
+set(CPM_HAS_CACHE_LOCK OFF)
+
```

Comment:
I think this has no implications, as it sets a local variable that is removed once the CPM.cmake file has finished evaluating. Perhaps it can be removed?

Reply:
Yes this can be removed.